src/app/change-account-selector/change-account-selector.component.html

```diff
@@ -1,29 +1,28 @@
-<div
-  class="fs-14px fc-default px-5px cursor-pointer change-account-selector br-4px change-account-selector__hover"
-  [ngClass]="{ 'change-account-selector__shadow': selectorOpen }"
-  #changeAccountSelectorRoot
-  id="changeAccountSelectorRoot"
->
-  <!-- Selector (unopened state) -->
-  <div class="d-flex align-items-center justify-content-between pl-5px pr-5px">
-    <div *ngIf="globalVars.loggedInUser" class="change-account-selector__ellipsis-restriction d-flex flex-row align-items-center cursor-pointer">
-      <div class="feed-create-post__avatar align-items-center change-account-selector__account-image" [avatar]="globalVars.loggedInUser.PublicKeyBase58Check"></div>
-      <div class="change-account-selector__acount-name change-account-selector__ellipsis-restriction">
-      {{
-        globalVars.loggedInUser.ProfileEntryResponse?.Username
-          ?  globalVars.loggedInUser.ProfileEntryResponse.Username
-          : globalVars.loggedInUser.PublicKeyBase58Check
-      }}</div>
-    </div>
-    <div
-      *ngIf="!globalVars.loggedInUser"
-      class="font-italic change-account-selector__ellipsis-restriction cursor-pointer"
-    >
-      Logged out
+<div #changeAccountSelectorRoot  id="changeAccountSelectorRoot">
+  <div
+    class="fs-14px fc-default px-5px cursor-pointer change-account-selector br-4px change-account-selector__hover"
+    [ngClass]="{ 'change-account-selector__shadow': selectorOpen }"
+  >
```

Comment:
Moving `#changeAccountSelectorRoot id="changeAccountSelectorRoot"` to the outer div causes #285. If you move it back into one div it works again.